### ISSUE TITLE
Everything works well except for P2 mouse movement

### ISSUE THREAD

**Issue by Danteb132** [kleirof/Coop_View#2]
I have a second monitor streaming Parsec to my friend and the movement and the dual cameras seem to be working perfectly but when my friend moves his mouse around, the direction in which his character is aiming is completely different. It gets more consistent if he aims far away from the character, but most of where you'd be aiming normally is totally off from where the game cursor actually is. Is it something with the PC cursor? If so I'm having trouble figuring that out. Thanks.

**Comment by kleirof** [comment]
Well I'll check it these days. And could you please offer some information about your gaming before my work? Like, what's your EtG main window's resolution? And your second window's resolution. Are your two windows fullscreen or not? Did you switch a option named 'second window startup resolution' in the run you discovered the issue, and what's its option value? This would help me locate the problem.

**Comment by Danteb132** [comment]
Main window res: 2560 x 1440
Second window res: 1920 x 1080

The two windows were not fullscreened because when I would fullscreen my main window, my mouse would stop working and even the system cursor would get stuck in the middle of the screen, which led me to have to Alt-F4 the game. I do think this was caused by fullscreening my main window but I'm not completely sure.

I hadn't tried that option for the second window startup resolution. I may be able to tinker with that option later and get back to you.

**Comment by kleirof** [comment]
A patched version has been released. You can see if it solves the aiming problem. Also fullscreen will  block mouse input somehow, it's temporarily banned. Please use borderless instead.

**Comment by Danteb132** [comment]
Sorry I forgot to reply. I did in fact use the Second window resolution setting and set it to 1920x1080, but borderless wouldn’t work properly as the aiming was still off. However, when I set my main gungeon to 1920x1080 windowed it actually
worked fine for my friend with no issue. If i stretched the window any bigger it bugged the aiming out for my friend but it did work. Not remotely a big deal since the mod is wonderful and a great experience. Thanks for your help!
